pkg/k8s/statedb_test.go

```diff
@@ -30,8 +32,11 @@ func ExampleRegisterReflector() {
 		"Reflector example",
 
 		cell.ProvidePrivate(
-			// The table the reflector writes to (RWTable[*Node]).
-			newTestNodeTable,
+			// Construct the table we're reflecting to.
+			func() statedb.RWTable[*corev1.Node] {
+				// statedb.NewTable(...)
```

Comment:
Should this be commented? Feels like it either should be uncommented or not present?

Reply:
Good point. I better fill it in for a proper example.